tests/python/test_fields_builder.py

```diff
@@ -3,174 +3,205 @@
 
 import taichi as ti
 
+'''
+Test fields with shape.
+'''
```

Comment:
These comments seem redundant.

Reply:
fixed

Reply:
Did you push the correct version? It seems that these lines are still there :-)

Reply:
Fixed and I removed all redundant notes.